apps/designer/remix.config.js

```diff
@@ -17,4 +17,7 @@ module.exports = {
   // assetsBuildDirectory: "public/build",
   serverBuildPath: "api/index.js",
   // publicPath: "/build/",
+  watchPaths: () => {
```

Comment:
This is the same as [here](https://github.com/webstudio-is/webstudio-designer/pull/163/files#diff-c6a539315bf0a259f65d6b284bc4d692549ea4a5287e04b73d6ca0daad96a503).